x-pack/plugins/triggers_actions_ui/public/application/lib/action_variables.ts

```diff
@@ -61,66 +67,93 @@ export enum LegacyAlertProvidedActionVariables {
   spaceId = 'spaceId',
 }
 
-function prefixKeys(actionVariables: ActionVariable[], prefix: string): ActionVariable[] {
-  return actionVariables.map((actionVariable) => {
-    return { ...actionVariable, name: `${prefix}${actionVariable.name}` };
-  });
+export enum SummaryAlertProvidedActionVariables {
+  ruleParams = 'rule.params',
+  newAlertsCount = 'alerts.new.count',
+  newAlertsData = 'alerts.new.data',
+  ongoingAlertsCount = 'alerts.ongoing.count',
+  ongoingAlertsData = 'alerts.ongoing.data',
+  recoveredAlertsCount = 'alerts.recovered.count',
+  recoveredAlertsData = 'alerts.recovered.data',
```

Comment:
We also have allAlertsData as well.
```
  allAlertsCount = 'alerts.all.count',
  allAlertsData = 'alerts.all.data',
```
And we need to add the available data under `data`. It's an array of objects and returns `alert-as-data` fields.
There is list for it on the below page, but i think it's not complete. I'll try to find the whole list.
https://github.com/elastic/kibana/blob/main/x-pack/plugins/rule_registry/README.md

Reply:
Added the allAlertsData variables in this commit affdbbe3ebf2fefe064cdacbcc98daa44b128beb
Created a follow on task for adding the data under `data`